decidim-elections/spec/system/election_log_spec.rb

```diff
@@ -0,0 +1,185 @@
+    let(:election) { create(:election, :bb_test, :key_ceremony_ended, component: component) }
+
+    it "shows that key ceremony is ended" do
+      expect(page).to have_content("The election got created and is successfully set up on the Bulletin Board.")
+      expect(page).to have_content("The key ceremony is completed. Every trustee has valid keys and has downloaded the necessary backup keys.")
+      expect(page).to have_content("The voting process has not started yet.")
+      expect(page).to have_content("The tally process has not started yet.")
+      expect(page).to have_content("The results are not published yet.")
+      expect(page).to have_content("The chained Hash of this message")
+    end
+  end
+
+  context "when voting has started" do
+    let(:election) { create(:election, :bb_test, :vote, component: component) }
+
+    it "shows that vote has started" do
+      expect(page).to have_content("The election got created and is successfully set up on the Bulletin Board.")
+      expect(page).to have_content("The key ceremony is completed. Every trustee has valid keys and has downloaded the necessary backup keys.")
+      expect(page).to have_content("The voting process has started.")
+      expect(page).to have_content("The tally process has not started yet.")
+      expect(page).to have_content("The results are not published yet.")
+      expect(page).to have_content("The chained Hash of this message")
+    end
+  end
+
+  context "when voting has ended" do
+    let(:election) { create(:election, :bb_test, :vote_ended, component: component) }
+
+    it "shows that voting process has ended" do
+      expect(page).to have_content("The election got created and is successfully set up on the Bulletin Board.")
+      expect(page).to have_content("The key ceremony is completed. Every trustee has valid keys and has downloaded the necessary backup keys.")
+      expect(page).to have_content("The voting process is finished.")
+      expect(page).to have_content("The tally process has not started yet.")
+      expect(page).to have_content("The results are not published yet.")
+      expect(page).to have_content("The chained Hash of this message")
+    end
+  end
+
+  context "when tally has started" do
+    let(:election) { create(:election, :bb_test, :tally, component: component) }
+
+    it "shows that tally has started" do
+      expect(page).to have_content("The election got created and is successfully set up on the Bulletin Board.")
+      expect(page).to have_content("The key ceremony is completed. Every trustee has valid keys and has downloaded the necessary backup keys.")
+      expect(page).to have_content("The voting process is finished.")
+      expect(page).to have_content("The tally process has started.")
+      expect(page).to have_content("The results are not published yet.")
+      expect(page).to have_content("The chained Hash of this message")
+    end
+  end
+
+  # context "when tally is completed" do
```

Comment:
💬 Are you still having problems with this one?

Reply:
Yeah. I'm on that one. 😄